extension/content.js

```diff
@@ -1,11 +1,12 @@
-let constructPosition = require('./lib/position.js').default;
+let $ = require('jquery');
+let position = require('./lib/position.js').default;
 
 chrome.runtime.onMessage.addListener(
   function(request, sender, sendResponse) {
     console.log('content.js', 'receive message');
 
     if (request.type === "capture_position") {
-      constructPosition(function(position) {
+      position.construct(function(position) {
```

Comment:
Eventually, we are probably going to rename the file `lib/position`, the function, `position` and its callback's argument, `position`.